tests/baselines/reference/commentsFunction.js

```diff
@@ -104,8 +104,8 @@ function foo2() {
 declare function foo(): void;
 /** This is comment for function signature*/
 declare function fooWithParameters(/** this is comment about a*/ a: string, 
-    /** this is comment for b*/
-    b: number): void;
+/** this is comment for b*/
+b: number): void;

```

Comment:
It seems like the indentation is incorrect here as well as in a number of other parameter lists in other files.

Reply:
Im pretty sure this indentation is consistent with our js emit?